exercises/poker/src/test/java/PokerTest.java

```diff
@@ -0,0 +1,162 @@
+import static org.junit.Assert.assertEquals;
+import java.util.Arrays;
+import org.junit.Ignore;
+import org.junit.Test;
+
+public class PokerTest
+{
```

Comment:
Thank you again for your suggestions. Sorry for taking so long time to answer, I wasn't available and  It's a little bit harder to fix so I need more time haha. I started but I think it's not enough for making a commit. I guess I will finish that this weekend.